src/ui/MobileMenu/ReactedMembersBottomSheet.tsx

```diff
@@ -28,8 +29,17 @@ export const ReactedMembersBottomSheet = ({
   emojiContainer,
 }: ReactedMembersBottomSheetProps): ReactElement => {
   const { members = [] } = channel;
+  const globalState = useSendbirdStateContext?.();
```

Comment:
@HoonBaek Is it safe to use `useSendbirdStateContext` in the `src/ui`?
I thought UI was an independent component element unrelated to business logic.

Reply:
Yes you are right. He was worried about this too. However, you can see that if `useSendbirdStateContext` is undefined, nothing happens (no error) so I think it is safe. If client wants this callback to work, they have to use `SendbirdProvider` too.

Reply:
`useSendbirdStateContext` is an imported function and therefore always exists. Although the return value of `useSendbirdStateContext()` may not exist, changing the behavior of `useSendbirdStateContext` can lead to side effects.

For example, in a recent update, I modified it to throw an error if there's no Provider.
[Link to the modification](https://github.com/sendbird/sendbird-uikit-react/blob/main/src/hooks/useSendbirdStateContext.tsx#L18)

If that's the policy(no business logic in ui), it seems like a good practice to continue following it for the sake of product maintenance and quality.

Would it be better to open the onPressUserProfileHandler interface in the UI?

```jsx
const EmojiReactions = ({ onPressUserProfile }) {
 return (<ReactedMembersBottomSheet onPressUserProfileHandler={onPressUserProfile} />)
}

const SomeModule = () => {
  const { eventHandlers } = useSendbirdStateContext();
  return (<EmojiReactions onPressUserProfileHandler={eventHandlers?.reaction?.onPressUserProfile} />)
}
```

Reply:
@bang9 that is good point. Let's not take this risk. Ill replace with onPressUserProfile